plugins/memblaze/memblaze-nvme.c

```diff
@@ -1616,7 +1616,7 @@ static int mb_get_smart_log_add(int argc, char **argv, struct command *cmd, stru
 
 	// Get log
 
-	struct smart_log_add log = {0};
+	struct smart_log_add log = {{{{{0}}}}};
```

Comment:
Uff, this looks rather unusual.  If this is really necessary because the compiler on CentOS can't handle this type of initialization, I suggest we rather use memset for this case.

Reply:
Changed to use memset as mentioned. Thank you.